sql/donut.sql

```diff
@@ -92,43 +90,41 @@ CREATE TABLE users (
     UNIQUE (username)
 );
 
--- Organizations Table
--- Organizations on campus. Used for positions, etc.
--- Examples: Blacker Hovse, IHC, CRC
-CREATE TABLE organizations (
-    org_id   INT         NOT NULL AUTO_INCREMENT,
-    org_name VARCHAR(32) NOT NULL,
-    type     VARCHAR(32) DEFAULT NULL,
-    PRIMARY KEY (org_id),
-    UNIQUE (org_name)
+-- Groups Table
+-- Groups of members. Used for mailing lists, etc.
+-- Groups are used for any internal/external groupings of users.
+-- Used for organizations on campus
+-- Examples: Blacker Hovse, IHC, CRC, ug-list, ug-2020
+CREATE TABLE groups (
+    group_id                INT          NOT NULL AUTO_INCREMENT,
+    group_name              VARCHAR(32)  NOT NULL,
+    group_desc              VARCHAR(255) DEFAULT NULL,
+    type                    ENUM('house','committee','ascit','publication'
```

Comment:
@majesticfish @RobertEng We should make a style guide for our SQL conventions, especially since
@joshc and I will be designing the vote module. https://github.com/ASCIT/donut/wiki/Donut-SQL-Conventions
For enums we can do this and back it up with a Python enum to make it readable for both backend code and from command line SQL. We can also use a numeric Python enum and a TINYINT, and manage the enum in Python. This sacrifices command line SQL readability for being able to add new enum values easily (for example if a new type of group is needed).